### ISSUE TITLE
Missing Square Brackets 

### ISSUE THREAD

**Issue by montellasebastien** [barshana-banerjee/ParaQA#2]
Hi! 

Thank you for delivering such verbalized dataset !

However, I have noticed that some verbalizations come with missing square brackets around the raw answer.  Overall, those samples only stand for 6.2% of the training set and 5.7% of the test set.
As an example, the training example with ***uid*** 342 has its second verbalized form with missing answer delimiters. If some raw answer could be found back from other verbalized forms, some verbalizations of same example possess different raw answers which leads to inconsistency.

A list of concerned uids:
[missing_square_brackets_train.txt](https://github.com/barshana-banerjee/ParaQA/files/6822931/missing_square_brackets_train.txt)
[missing_square_brackets_test.txt](https://github.com/barshana-banerjee/ParaQA/files/6822930/missing_square_brackets_test.txt)

Do you plan to correct that in future ? Do you accept any PR ?

Thank you !
